source/FreeRTOS_Stream_Buffer.c

```diff
@@ -381,19 +355,20 @@ size_t uxStreamBufferGet( StreamBuffer_t * pxBuffer,
     }
 
     /* Use the minimum of the wanted bytes and the available bytes. */
-    uxCount = FreeRTOS_min_size_t( uxSize, uxMaxCount );
+    uxByteCount = FreeRTOS_min_size_t( uxSize, uxByteCount );
```

Comment:
Same remark as earlier, please do not change the value of the function parameter `uxByteCount`.

Reply:
So this relates to what I was asking @ActoryOu, is there any reason to have const written for pass by value parameters? For example, there is 
`size_t uxStreamBufferSpace( const StreamBuffer_t * const pxBuffer,
                            const size_t uxLower,
                            const size_t uxUpper );`
but if the uxLower & uxUpper parameters can't be changed anyways due to MISRA, then I suppose there is no value in having the const for uxLower and uxUpper? I'm just trying to keep things consistent as best as possible.

Reply:
You wrote:

> but if the uxLower & uxUpper parameters can't be changed anyways due to MISRA, then I suppose there is no value in having the const for uxLower and uxUpper?

I agree that it doesn't add much value to add the const keyword to parameters that are passed by-value. But it doesn't hurt either.